IPS/IPS-IOC-01App/Db/scpi_system_alarms.db

```diff
@@ -0,0 +1,235 @@
+# Macro             Default
+# BOARDID_MAG       MB1.T1
+# BOARDID_10TMAG    DB8.T1
+# BOARDID_PRESS     DB5.P1
+# BOARDID_LEVEL     DB1.L1
+#
+# Poll system alarms: This record repeatedly sends the READ:SYS:ALRM command
+# and the response is read as a string and parsed by the aSub record below.
+#
+record(waveform, "$(P)STS:_POLLSYSTEM:ALARMS") {
+  field(DESC, "Poll system alarms")
+  field(DTYP, "stream")
+  field(FTVL, "CHAR")
+  field(NELM, "1000")  # Adjust size as needed
+  field(SCAN, "5 second")
+  field(INP, "@OxInstIPS_SCPI.protocol getSysAlarms $(PORT)")
+  field(FLNK, "$(P)STS:_POLLSYSTEM:ALARMS_SUB")
+
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:STS:_POLLSYSTEM:ALARMS")
+  field(SDIS, "$(P)DISABLE")
+}
+
+# The following stringout records are used to provide the board identifiers
+record(stringout, "$(P)STS:SYSTEM:ALARMS:_NAME:TBOARD") {
+  field(DESC, "Temperature board UID name")
+  field(DTYP, "Soft Channel")
+  field(SCAN, "Passive")
+  field(PINI, "YES")
+  field(VAL, "$(BOARDID_MAG)")
+  field(OUT, "")
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:STS:SYSTEM:ALARMS:_NAME:TBOARD")
+  field(SDIS, "$(P)DISABLE")
+}
+record(stringout, "$(P)STS:SYSTEM:ALARMS:_NAME:10TBOARD") {
+  field(DESC, "Temperature 10T board UID name")
+  field(DTYP, "Soft Channel")
+  field(SCAN, "Passive")
+  field(PINI, "YES")
+  field(VAL, "$(BOARDID_10TMAG)")
+  field(OUT, "")
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:STS:SYSTEM:ALARMS:_NAME:10TBOARD")
+  field(SDIS, "$(P)DISABLE")
+}
+record(stringout, "$(P)STS:SYSTEM:ALARMS:_NAME:LBOARD") {
+  field(DESC, "Level board UID name")
+  field(DTYP, "Soft Channel")
+  field(SCAN, "Passive")
+  field(PINI, "YES")
+  field(VAL, "$(BOARDID_LEVEL)")
+  field(OUT, "")
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:STS:SYSTEM:ALARMS:_NAME:LTBOARD")
+  field(SDIS, "$(P)DISABLE")
+}
+record(stringout, "$(P)STS:SYSTEM:ALARMS:_NAME:PBOARD") {
+  field(DESC, "Pressure board UID name")
+  field(DTYP, "Soft Channel")
+  field(SCAN, "Passive")
+  field(PINI, "YES")
+  field(VAL, "$(BOARDID_PRESS)")
+  field(OUT, "")
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:STS:SYSTEM:ALARMS:_NAME:PBOARD")
+  field(SDIS, "$(P)DISABLE")
+}
+
+# This record is triggered by the stringin record above and parses the response.
+# It extracts the board identifier and the alarm message, and writes them to the appropriate
+# output fields. The output fields are then used by the mbbi and bi records below to set the
+# alarm status for each board.
+# Board identifiers are provided as macros and passed to the aSub as input fields B-E.
+#
+record(aSub, "$(P)STS:_POLLSYSTEM:ALARMS_SUB") {
+  field(SNAM, "handle_system_alarm_status")
+  field(INPA, "$(P)STS:_POLLSYSTEM:ALARMS NPP NMS")
+#  field(INPB, "$(P):STS:SYSTEM:ALARMS:_NAME:TBOARD")
+#  field(INPC, "$(P):STS:SYSTEM:ALARMS:_NAME:10TBOARD")
+#  field(INPD, "$(P):STS:SYSTEM:ALARMS:_NAME:LBOARD")
+#  field(INPE, "$(P):STS:SYSTEM:ALARMS:_NAME:PBOARD")
```

Comment:
If these are unused, can they be removed - along with the `FT<n>`, `NO<n>` defs?